js/src/forum/utils/slidable.js

```diff
@@ -36,7 +36,7 @@ export default function slidable(element) {
     // bit of a workaround. We set up the animation with a step function that
     // will set the transform property, but then we animate an unused property
     // (background-position-x) with jQuery.
-    options.duration = options.duration || 'fast';
+    options.duration ||= 200;
```

Comment:
Do we need to change it to a magic number? If so, let's put it in a constant

Reply:
It's just as magic as the word `fast`, which acts as a synonym for `200` internally within jQuery.

Reply:
Sure, but the string `fast` is descriptive and feels less magic than `200`.

Reply:
```suggestion
    options.duration ||= 'fast';
```